test/org/zaproxy/zap/extension/ascanrulesAlpha/TestNoSQLInjectionUnitTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2018 The ZAP Development Team
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ */
+package org.zaproxy.zap.extension.ascanrulesAlpha;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+import org.zaproxy.zap.model.Tech;
+import org.zaproxy.zap.model.TechSet;
+
+
+/**
+ * Unit test for {@link TestNoSQLInjection}.
+ */
+public class TestNoSQLInjectionUnitTest extends ActiveScannerTest<TestNoSQLInjection> {
+
+    @Override
+    protected TestNoSQLInjection createScanner() {
+        return new TestNoSQLInjection();
+    }
+
+    @Test
+    public void shouldTargetDbTech() {
+        // Given
+        TechSet techSet = techSet(Tech.Db);
+        // When
+        boolean targets = rule.targets(techSet);
+        // Then
+        assertThat(targets, is(equalTo(true)));
+    }
+    
+    @Test
+    public void shouldTargetMongoDBTech() {
+        // Given
+        TechSet techSet = techSet(Tech.MongoDB);

```

Comment:
As noted above, use of new core components that are not yet in this repo will fail.